docs/docs/03-guides/01-first-tool.mdx

```diff
@@ -0,0 +1,29 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import PythonCryptoToolReadme from '../../snippets/python-crypto-tool-readme.md.mdx';
+import NodeCryptoToolReadme from '../../snippets/node-crypto-tool-readme.md.mdx';
+import GoCryptoToolReadme from '../../snippets/go-crypto-tool-readme.md.mdx';
```

Comment:
These files are generated by fetching the README.md from each tool's GitHub repo. This the docusaurus does this automatically at build time.